torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -289,13 +289,9 @@ class FullyShardedDataParallel(nn.Module):
             off between memory saving and communication overhead. ``FULL_SHARD``
             will be chosen if sharding_strategy is not specified.
         cpu_offload (Optional[CPUOffload]):
-            CPU offloading config. Currently, only parameter and gradient CPU
-            offload is supported. It can be enabled via passing in
-            ``cpu_offload=CPUOffload(offload_params=True)``. Note that this
-            currently implicitly enables gradient offloading to CPU in order for
-            params and grads to be on same device to work with optimizer. This
-            API is subject to change. Default is ``None`` in which case there
-            will be no offloading.
+            This configures CPU offloading. If this is set to ``None``, then
```

Comment:
shall we mention parameter and gradient CPU offload, as users might thought this is for activation.

Reply:
Sorry, I missed this comment, and CI just turned green. Let me do this in a follow-up.

Reply:
Actually, I think it is not a huge deal because if the user clicks on the `CPUOffload` link, then the user will see that it is only for parameters and gradients.

Reply:
It might be harder to have two sources of truth (i.e. both here and also in the `CPUOffload` docs).